transcom/mymove
# MAIN B-20104, B-20105, B-20106 remove Form/WizardForm, SignatureForm, and AccountingPanel

> [!NOTE]  
> This `main` PR contains three different `integrationTesting` PRs to avoid excessive builds in CircleCI since it is 100% deletion of tech debt and no added functionality

[B-20104 INTEGRATION PR](https://github.com/transcom/mymove/pull/13044)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20104)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `WizardFormPage` component and its use of `redux-form`.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and introducing it to meet baby jeebus should complete this task.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `WizardFormPage` component
3. Confirm that `WizardFormPage` no longer exists in the code base and that it isn't being used anywhere else


<br><br>
[B-20105 INTEGRATION PR](https://github.com/transcom/mymove/pull/13041)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20105)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `SignatureForm` component and its use of `redux-form` as well as accessory files.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and sending it to the heavens completes this task and should NEVER BE SPOKEN ABOUT AGAIN.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `SignatureForm` component
3. Confirm that `SignatureForm` no longer exists in the code base and that it isn't being used anywhere else


<br><br>
[B-20106 INTEGRATION PR](https://github.com/transcom/mymove/pull/13042)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20106)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `AccountingPanel` component and its use of `redux-form` as well as accessory files.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and sending it to outer space should complete the task.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `AccountingPanel` component
3. Confirm that `AccountingPanel` no longer exists in the code base and that it isn't being used anywhere else
